Content.Server/Tiles/FloorTileSystem.cs

```diff
@@ -35,10 +40,16 @@ private void OnAfterInteract(EntityUid uid, FloorTileComponent component, AfterI
 
             // this looks a bit sussy but it might be because it needs to be able to place off of grids and expand them
             var location = args.ClickLocation.AlignWithClosestGridTile();
+            foreach (var ent in location.GetEntitiesInTile())
```

Comment:
the system should resolve the `EntityLookupSystem` and pass it to this helper, or just not use the helper in the first place.